ui/src/app/components/webcomponents/element-form.component.ts

```diff
@@ -329,6 +329,20 @@ export class ElementFormComponent extends BaseComponent implements OnInit {
     this.chromeRecorderService.elementCallBack = undefined;
     this.chromeRecorderService.stopSpying();
     this.formDetails ? this.mobileRecorderEventService.setEmptyAction() : ( isClose? this.dialogRef.close(this.reviewSubmittedElement): null );
+    if(isClose && this.options.isStepRecordView){
+      let version=new WorkspaceVersion();
+      version.id=this.versionId;
+      let sendDetails = {
+        versionId: this.versionId,
+        version:version,
```

Comment:
does it require, to have version and versionId in two different prop? @manikanta-reddy-t

Reply:
updated